composer.json

```diff
@@ -19,16 +19,16 @@
 
     "require": {
         "php":                          ">=7.2.5",
-        "symfony/dependency-injection": "^4.3|^5.0|^6.0",
-        "symfony/translation":          "^4.3|^5.0|^6.0",
-        "symfony/config":               "^4.3|^5.0|^6.0"
+        "symfony/dependency-injection": "^5.4|^6.0",
+        "symfony/templating": "^5.4|^6.0",
+        "symfony/translation":          "^5.4|^6.0",
+        "symfony/config":               "^5.4|^6.0"
```

Comment:
we have alignment in some places but not others - let's ditch the alignment everywhere